crates/db_schema/src/utils.rs

```diff
@@ -520,12 +514,6 @@ pub fn build_db_pool_for_tests() -> ActualDbPool {
   build_db_pool().expect("db pool missing")
 }
 
-#[allow(clippy::expect_used)]
-static EMAIL_REGEX: LazyLock<Regex> = LazyLock::new(|| {
-  Regex::new(r"^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$")
-    .expect("compile email regex")
-});
```

Comment:
Same here, we're not validating emails anymore?

Reply:
It seems not, there is only a check that the email is not taken yet. Must be a PR somewhere but I cant find it. The regex probably failed in some cases, and anyway there is the confirmation email to ensure that it works.